crt_portal/cts_forms/models.py

```diff
@@ -159,5 +159,8 @@ def assign_section(self):
                 return 'IER'
             elif 'Disability (including temporary or recovery)' not in protected_classes:
                 return 'ELS'
+        elif self.primary_complaint == 'commercial_or_public':
```

Comment:
I think we might be missing the routing to `HCE` when the primary complaint is specifically housing? That is the first routing criterium in the issue and I don't _think_ I see it already in this file.

Reply:
Haha- I will add the main use case ( ^ ᗜ ^ )